enterprise/__init__.py

```diff
@@ -2,6 +2,6 @@
 Your project description goes here.
 """
 
-__version__ = "3.21.3"
+__version__ = "3.21.4"
```

Comment:
See https://open-edx-proposals.readthedocs.io/en/latest/oep-0047-bp-semantic-versioning.html to understand a better version update.

Reply:
Done